cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs

```diff
@@ -112,7 +112,7 @@ hooks =
   , ("git-annex >= 6.20170925 && < 6.20171214", set doCheck False)      -- some versions of git-annex require their test suite to be run inside of a git checkout
   , ("github-backup", set (executableDepends . tool . contains (pkg "git")) True)
   , ("GLFW", over (libraryDepends . system) (Set.union (Set.fromList [bind "pkgs.xorg.libXext", bind "pkgs.xorg.libXfixes"])))
-  , ("GlomeVec", set (libraryDepends . pkgconfig . contains (bind "self.llvmPackages.llvm")) True)
+  , ("GlomeVec", set (libraryDepends . pkgconfig . contains (bind "pkgs.llvmPackages.llvm")) True)
```

Comment:
This package passes `-fllvm`, so this is actually correct and necessary to bring the LLVM tools into scope.